vslm/client.go

```diff
@@ -55,5 +56,15 @@ func NewClient(ctx context.Context, c *vim25.Client) (*Client, error) {
 		return nil, err
 	}
 
-	return &Client{sc, res.Returnval}, nil
+	return &Client{c, sc, res.Returnval}, nil
+}
+
+// RegisterVolume calls the RegisterDisk method in globalObjectManager to register virtual disks as First Class Disks
```

Comment:
This method isn't doing anything specific to volumes, but is a helper/shortcut for global RegisterDisk, right?  Maybe it should live in the cns package instead?

Reply:
+1
This method is not needed at all